centaur/src/main/resources/standardTestCases/workflow_engine_functions.test

```diff
@@ -0,0 +1,13 @@
+name: workflow_engine_functions
+testFormat: workflowsuccess
+
+files {
+  workflow: workflow_engine_functions/workflow_engine_functions.wdl
+}
+
+metadata {
+  "outputs.w.hello": "hello"
+  "outputs.w.bar": "bar"
+  "outputs.w.world": "world"
+  "outputs.w.qux": "qux"
```

Comment:
Could these be written in the same order that they would be produced? ie `hello`, `world`, `bar`, `qux`?

Reply:
sure